.circleci/config.yml

```diff
@@ -121,7 +124,23 @@ jobs:
               echo "Running >> npm publish"
               npm publish
             fi
-
+#  api-governance:
+#    machine: true
+#    working_directory: ~/project
+#    steps:
+#      - checkout
+#      - attach_workspace:
+#          at: ~/project
+#      - run:
+#          name: Login for Docker
+#          command: echo $DOCKER_TOKEN | docker login -u "$DOCKER_USER" --password-stdin
+#      - run:
+#          name: API Governance Compliance
+#          command: |
+#            docker pull qlik/asmquery
+#            docker create -v /specs --name specs alpine:3.4 /bin/true
+#            docker cp ./api-specifications/properties.json specs:/specs
+#            docker run --volumes-from specs -e target=Colony -e runMode=failBuild -e apiId=abdc47c2-921a-45d3-ba5c-80d448574c5b -e specPath='/specs/spec.json' -e commit="$CIRCLE_SHA1" qlik/asmquery
```

Comment:
Should this be uncommented later at some point?

Reply:
yea, but need to get it registred in api machinery first.